backend/config/connectDB.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const mongoURI = process.env.mongoURI;
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect(mongoURI, {
+      useNewUrlParser: true,
+      useCreateIndex: true,
+      useFindAndModify: false
```

Comment:
Is the useFindAndModify parameter normally set to true by default when connecting mongoose (I did a brief look up of it in the mongoose docs)?  Are we setting it to false to modify  search queries?

Reply:
Not exactly sure what this is exactly. The only thing I know is that without these 3, it still works normally. The only difference is that it would give me a warning in the console about something being deprecated and for us to add those 3 in the connect method to remove the warning. 

I'm assuming that later on, Mongoose will update it so that it will not need those to be included in the connect method.

Reply:
Gotcha, thank you for clarifying! I was just curious about the parameter settings. I looked up the useFindAndModfy in the documentation,  it says default set to true true unless you were setting other methods as the native. Was curious about the reasoning. :-)